File: task2/task2.c
/*
 * task2.c
 *
 *  Created on: 27 дек. 2016 г.
 *      Author: Tanyushka
 */

#include <stdio.h>
#define TRUE 1

int main(void){
	int current = 1;
	int previous = 0;
	int temp;
	int sum = 0;
	while(TRUE){
		if((current + previous) > 4000000)
			break;
		if(current%2 == 0){
			sum += current;
		}
		temp = current;
		current += previous;
		previous = temp;
	}
	printf("%d, %d", sum, current);
	return 0;
}

File: task2/Task2.java
class Task2 {
    public static void main(String args[])
    {
        int current = 1;
        int previous = 0;
        int summary = 0;
        int temp = 0;
        while (current + previous <= 4000000)
        {
            temp = current;
            current += previous;
            previous = temp;
            if (current%2 == 0) summary += current;
        }
        System.out.println(summary);
    }
}